packages/react-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/react-components",
-  "version": "10.0.0",
+  "version": "11.0.0",
```

Comment:
Since this is a `major` breaking change, I think this would be a good PR to create a component-library release after it's merged (so `core` would need a version bump too). 

With that said, let's hold off on merging this PR right away until we are ready to create a new release (there are a couple of other PRs that I think we might want to get in first).